javascript/index.js

```diff
@@ -1,30 +1,39 @@
-const chronometer = new Chronometer();
+let chronometer = new Chronometer();
+
+
+let btnLeft = document.getElementById('btnLeft');
 
-// get the buttons:
-const btnLeft = document.getElementById('btnLeft');
-const btnRight = document.getElementById('btnRight');
 
-// get the DOM elements that will serve us to display the time:
 let minDec = document.getElementById('minDec');
 let minUni = document.getElementById('minUni');
+
 let secDec = document.getElementById('secDec');
 let secUni = document.getElementById('secUni');
-let milDec = document.getElementById('milDec');
-let milUni = document.getElementById('milUni');
-let splits = document.getElementById('splits');
 
 function printTime() {
   // ... your code goes here
+  //callbacks of startClick prints the time, so you have to pass this when calling the startClick function
+
+  printMinutes();
+  printSeconds();
+  printMilliseconds();
 }
 
 function printMinutes() {
   // ... your code goes here
+  let min = chronometer.twoDigitsNumber(chronometer.getMinutes());
```

Comment:
twoDigitsNumber will return a number if the number has two digits, so you won't be able to use bracket notation on numbers.